core-bundle/src/EventListener/ExceptionConverterListener.php

```diff
@@ -51,6 +51,7 @@ class ExceptionConverterListener
         PageNotFoundException::class => 'NotFoundHttpException',
         ServiceUnavailableException::class => 'ServiceUnavailableHttpException',
         ContaoServiceUnavailableException::class => 'ServiceUnavailableHttpException',
+        \UnusedArgumentsException::class => 'NotFoundHttpException',
```

Comment:
Should we move the `UnusedArgumentsException` to the `Contao` namespace?

Reply:
I think we shouldn‘t change that. It‘s legacy anyway and won‘t make it to 5.0 :)

Reply:
I thought about this too, but decided against it for the same reason.